### PR TITLE
#8: Support for deleting items

### PR DESCRIPTION
**Pull request by pandreev**
please review

### CODE REVIEW

**Comment by dmarkov** [comment]
Thanks, someone will review your pull request soon

**Comment by dmarkov** [comment]
@yegor256 review this one please

File: src/main/java/com/jcabi/dynamo/AwsItem.java [hunk 1]
```diff
@@ -215,6 +217,28 @@ public AttributeValue get(@NotNull(message = "attribute name can't be NULL")
...
+        final AmazonDynamoDB aws = this.credentials.aws();
+        try {
+            final DeleteItemRequest request = new DeleteItemRequest();
+            request.setTableName(this.name);
+            request.setKey(this.attributes.only(this.keys));
+            request.setReturnConsumedCapacity(ReturnConsumedCapacity.TOTAL);
+            final long start = System.currentTimeMillis();
+            final DeleteItemResult result = aws.deleteItem(request);
+            Logger.info(
+                    this, "#remove('%s'): removed item from DynamoDB%s, in %[ms]s",
```
Comment (yegor256) [inline src/main/java/com/jcabi/dynamo/AwsItem.java]: why 8 indentation spaces?

File: src/main/java/com/jcabi/dynamo/Item.java [hunk 2]
```diff
@@ -107,6 +107,14 @@ boolean has(@NotNull(message = "attribute name can't be NULL") String name)
         Map<String, AttributeValueUpdate> attrs) throws IOException;
 
     /**
+     * Remove item
+     *
+     * @throws IOException In case of DynamoDB failure
+     */
+    void remove()
+            throws IOException;
```
Comment (yegor256) [inline src/main/java/com/jcabi/dynamo/Item.java]: hm.. why new line here?

**Comment by yegor256** [comment]
@pandreev a few small comments above. And here is a big one - where is a unit test? Without a test, how do you/we know that it works?

**Comment by pandreev** [comment]
@yegor256 small comments are fixed. I could not find any unit tests for Item and its methods, that is why a unit test for remove is missing. Please let me know where to add a unit tests and will take care to be added.

**Comment by yegor256** [comment]
@pandreev just create an `AwsItemTest` and write a test there, as usual. Or I didn't understand the question?

**Comment by pandreev** [comment]
@yegor256 Yes, that was exactly my question. As new to the projects, I didn't know what is the way of working and when I saw that unit tests are missing for Item, I assumed that this might be separated to another task. No problem, will create AwsItemTest and add a unit tests for remove().

**Comment by yegor256** [comment]
@pandreev are you planning to finish this pull request? if not, please close it

**Comment by pandreev** [comment]
Sorry, no Internet at home last two months.

**Comment by dmarkov** [comment]
@yegor256 thank you, added **18 mins** to your acc, payment num is `000-bc271c89`

you're getting extra minutes here (c=3)

+18 to your rating, your total score is [+7412](http://www.netbout.com/b/35008?open=rating)

**Pull request closed without merging**